source/plugins/ruby/KubernetesApiClient.rb

```diff
@@ -517,7 +517,7 @@ def getContainerResourceRequestsAndLimits(pod, metricCategory, metricNameToColle
               #No container level limit for the given metric, so default to node level limit
             else
               if (metricCategory == "limits" && !nodeAllocatableRecord.nil? && !nodeAllocatableRecord.empty? && nodeAllocatableRecord.has_key?(metricNameToCollect))
-                metricValue = nodeAllocatableRecord[metricNameToCollect]
+                metricValue = getMetricNumericValue(metricNameToCollect, nodeAllocatableRecord[metricNameToCollect])
```

Comment:
kaveesh, can you please search for places where this function is called (getMetricNumericValue) to see if there are any more occurences ?

Reply:
Found one other location, added it there.